doc/release-notes/SONiC_202405_Release_Notes.md

```diff
@@ -0,0 +1,109 @@
+# SONiC 202405 Release Notes
+
+This document captures the new features added and enhancements done on existing features/sub-features for the SONiC [202405](https://github.com/orgs/sonic-net/projects/17/) release.
+
+
+
+# Table of Contents
+
+ * [Branch and Image Location](#branch-and-image-location)
+ * [Dependency Version](#dependency-version)
+ * [Security Updates](#security-updates)
+ * [Feature List](#feature-list)
+ * [SAI APIs](#sai-apis)
+ * [Contributors](#contributors)
+
+
+# Branch and Image Location  
+
+Branch : https://github.com/Azure/sonic-buildimage/tree/202405 <br> 
+Image  : https://sonic-build.azurewebsites.net/ui/sonic/pipelines  (Example - Image for Broadcom based platforms is [here](https://sonic-build.azurewebsites.net/ui/sonic/pipelines/138/builds/51255/artifacts/98637?branchName=master&artifactName=sonic-buildimage.broadcom))
+
+# Dependency Version
+
+|Feature                    | Version  |
+| ------------------------- | --------------- |
+| Linux kernel version      | linux_6.1.38-4  |
+| SAI   version             | SAI v1.14.0    |
+| FRR                       | 8.5.4   |
+| LLDPD                     | 1.0.16-1+deb12u1 |
+| TeamD                     | 1.30-1    |
+| SNMPD                     | 5.9+dfsg-4+deb11u1    |
+| Python                    | 3.11.2-6    |
+| syncd                     | 1.0.0    |
+| swss                      | 1.0.0    |
+| radvd                     | 2.18-3    |
+| isc-dhcp                  | 4.4.3-P1-2  |
+| sonic-telemetry           | 1.1    |
+| redis-server/ redis-tools | 5.0.3-3~bpo9+2    |
+| Debian version			| Migrated to Bookworm (Debian version 12)	|
+
+Note : The kernel version is migrated to the version that is mentioned in the first row in the above 'Dependency Version' table.
+
+
+# Security Updates
+
+1. Kernel upgraded from 5.10.136-1 to 6.1.38-4 for SONiC release.<br>
+   Change log: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.38
+
+2. Docker upgraded from  24.0.7-debian-stretch to 27.0.1-debian-stretch <br>
```

Comment:
Where is this coming from? The docker daemon is still on 24.0.2-1~debian.12~bookworm

Reply:
Addressed as suggested.